metricbeat/module/kubernetes/event/types.go

```diff
@@ -0,0 +1,46 @@
+package event
+
+import "time"
+
+type ObjectMeta struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ObjectMeta should have comment or be unexported